app/utils/clipboard.ts

```diff
@@ -34,15 +34,25 @@ export const showModalClipboardIfValidPayment = async ({
     return
   }
 
-  const { valid } = parsePaymentDestination({
+  const { valid, paymentType } = parsePaymentDestination({
     destination: clipboard,
     network,
     pubKey: myPubKey,
   })
 
-  if (!valid) {
+  if (!valid || paymentType === "intraledger") {
```

Comment:
Am I understanding this right - this means that if the user has a pay url copied e.g. https://ln.bitcoinbeach.com/unclesamtoshi then it won't trigger the modal?

Reply:
Yes. I will remove this line. Originally I thought it could be worth removing because the way that the galoy-client parses the string will return intraledger by default, so even if something is clearly not a username (such as your phone auth code) it will still trigger an invoice. However the better solution is to change the galoy-client to better parse the strings.